api/hardwaremanagement/v1alpha1/node.go

```diff
@@ -65,7 +65,7 @@ type NodeStatus struct {
 //
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +operator-sdk:csv:customresourcedefinitions:displayName="ORAN O2IMS Cluster Request",resources={{Namespace, v1}}
+// +operator-sdk:csv:customresourcedefinitions:displayName="ORAN O2IMS Provisioning Request",resources={{Namespace, v1}}
```

Comment:
I know this was wrong before, but I think the displayName should just be "Node" here.

Reply:
Done